bramkouwen/SVG-Icon-Marketplace
# Adding images

## What does this PR do?
* It adds the images folder
* It places SVGs in the images folder

## How do you check this PR?
* Pull down the branch
* See added images folder and content 

## Screenshots 
None.
